src/scenario.jl

```diff
@@ -381,8 +380,8 @@ function run_model(domain::Domain, param_set::NamedDimsArray, corals::DataFrame,
     seed_sc_CA::Int64 = first(findall(corymbose_enhanced .& target_class_id))
 
     # Extract colony areas for sites selected and convert to m^2
-    col_area_seed_TA = corals.colony_area_cm2[seed_sc_TA] / 10^4
-    col_area_seed_CA = corals.colony_area_cm2[seed_sc_CA] / 10^4
+    col_area_seed_TA = pi *(corals.mean_colony_diameter_m[seed_sc_TA]/2)^2
```

Comment:
Fix spacing here (did auto-format not work?).

Use consistent data types as well (e.g., `/2` should be `/ 2.0`)

Reply:
The same formula is used below (L 384) - might as well put it in a function for readability.

Reply:
I just made a function instead as you suggested and delleted slow_ODE. Let me know if there are other changes needed. Tests pass on my end.